src/test/java/com/n1analytics/paillier/PaillierEncryptedNumberTest.java

```diff
@@ -1051,14 +1051,14 @@ public void testCiphertextObfuscation() throws Exception {
     public void testNotObfuscated() throws Exception {
       EncryptedNumber encryptedNumber = context.encrypt(context.encode(3.14, 103));
       assertEquals(encryptedNumber.isSafe, false);
-      BigInteger ciphertext1 = encryptedNumber.ciphertext;
```

Comment:
similarly here, I think the intention was to access the ciphertext in its non-obfuscated form